# my_math.py
def my_factorial(n):
    if n < 0:
        raise ValueError("Невозможно вычислить факториал отрицательного числа")
    result = 1
    for i in range(2, n + 1):
        result *= i
    return result
# main.py
from my_math import my_factorial
if __name__ == "__main__":
    try:
        number = int(input("Введите целое неотрицательное число: "))
        print(f"Факториал числа {number}: {my_factorial(number)}")
    except ValueError as e:
        print(e)
